dtsync/cbor_gen.go

```diff
@@ -88,7 +93,7 @@ func (t *Message) UnmarshalCBOR(r io.Reader) error {
 		return fmt.Errorf("cbor input should be of type array")
 	}
 
-	if extra != 3 {
+	if extra != 4 {
```

Comment:
This will be a problem with messages coming from non-updated lotus, since they will only have 3 fields.